cli/flox/src/commands/services/status.rs

```diff
@@ -302,23 +298,33 @@ mod tests {
     }
 
     #[test]
-    fn test_processstatedisplay_json_lines() {
+    fn test_processstatesdisplay_json_array() {
         let states = ProcessStates::from(vec![
             generate_process_state("aaa", "Running", 123, true),
             generate_stopped_process_state("bbb", 456),
             generate_completed_process_state("ccc", 789, 0),
         ]);
         let states_display: ProcessStatesDisplay = states.into();
-        let mut buffer = Vec::new();
-        for proc in states_display {
-            let line = serde_json::to_string(&proc).unwrap();
-            writeln!(buffer, "{line}").unwrap();
-        }
-        let buffer_str = String::from_utf8(buffer).unwrap();
-        assert_eq!(buffer_str, indoc! {r#"
-            {"name":"aaa","status":"Running","pid":123,"exit_code":null}
-            {"name":"bbb","status":"Stopped","pid":456,"exit_code":null}
-            {"name":"ccc","status":"Completed","pid":789,"exit_code":0}
-        "#});
+        let json_array = serde_json::to_string_pretty(&states_display).unwrap();
+        assert_eq!(json_array, indoc! {r#"[
+              {
+                "name": "aaa",
```

Comment:
(non-blocking, commentary) In hindsight it would have been nice if this was `{"name": {"status": …}}` so that you could pull out entries with `jq '.["aaa"].status'` rather than `jq '.[] | select(.name == "aaa").status'` but that's a bit more work and it's a slightly more breaking change that we can always make later.